### DISCUSSION TITLE
Welcome to astro Discussions!

### DISCUSSION THREAD

**Discussion by dmcalester** [RocketCommunicationsInc/astro#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by huseyin-bozkurt** [comment]
Hello,
I would like use your framework to build a web site on Angular but it seems that there is no "@astrouxds/angular" package in npm.
I got the following error (I also checked the npm registry, there is no suck package there).
'@astrouxds/angular@latest' is not in the npm registry.
Regards.
Huseyin

**Reply by markacianfrani** [reply]
We're still working on testing the angular package before we officially release it. There are a few issues we need to work out before it'll be ready for production, but it should be coming very soon. I'll update this thread once we're ready to release it

**Reply by markacianfrani** [reply]
@huseyin-bozkurt the angular wrapper has been published. you can check out the documentation for that at https://astro-components.netlify.app/?path=/story/astro-uxds-welcome-angular--page

**Reply by huseyin-bozkurt** [reply]
@markacianfrani  
Thank you for notification.
Is timeline available in the angular package?
Thank you in advance.

**Reply by markacianfrani** [reply]
@huseyin-bozkurt not at the moment. [timeline is on our roadmap however ](https://github.com/RocketCommunicationsInc/astro/discussions/15)
